### PR TITLE
Upload nightly torch/torchvision wheels to our release page

### PR DESCRIPTION
**Pull request by mgehre-amd**

### CODE REVIEW

File: .github/workflows/buildRelease.yml [hunk 1]
```diff
@@ -67,6 +67,24 @@ jobs:
       with:
         release_id: ${{ github.event.inputs.release_id }}
         assets_path: ./build_tools/python_deploy/wheelhouse/torch*.whl
+
+    - name: Download nightly pytorch and torchvision
+      if: github.event.inputs.release_id != ''
+      run: |
+        mkdir deps
```
Comment (TinaAMD) [inline .github/workflows/buildRelease.yml:74]: Where is this `deps` folder created? How is this relating to the URL under which we can later on find the wheel?

Comment (mgehre-amd) [reply .github/workflows/buildRelease.yml:74]: It's created in the working directory of the github actions run, which I think is the same directory as the git working directory. This does not affect the URL of where the wheel is uploaded to afaik.

**Comment by mgehre-amd** [comment]
A build of this action is currently running here: https://github.com/Xilinx/torch-mlir/actions/runs/6093701020

**Review by TinaAMD — APPROVED** [review]

**Comment by mgehre-amd** [comment]
Releases are uploaded here: https://github.com/Xilinx/torch-mlir/releases/tag/snapshot-20230906.349

**Pull request merged**